### DISCUSSION TITLE
LinkedIn integration

### DISCUSSION THREAD

**Discussion by Sytten** [twentyhq/twenty#7429] [Ideas]
I would very much like to have a LinkedIn integration so it pulls the people with whom I interacted with + selected people in my contact network.

**Comment by mmaryo** [comment]
+1

**Comment by chsasank** [comment]
I hope the devs make it easy for us to write/host these integrations/plugins. Probably easy way is to write a separate service that does what we want by interacting with twenty's API. 

Here is some useful python project to do what you're looking for: https://github.com/tomquirk/linkedin-api?tab=readme-ov-file

**Comment by simsra** [comment]
+1

**Comment by toobulkeh** [comment]
We've done a lot of Linkedin integration work over @ Tarka. Happy to chat more about this (and we're working on going open source!)

**Reply by mezbahalam** [reply]
> We've done a lot of Linkedin integration work over @ Tarka. Happy to chat more about this (and we're working on going open source!)

Is your LinkedIn integration open-sourced yet?

**Reply by vincentmcleese** [reply]
@toobulkeh curious about this, i'm thinking about setting up a linkedin integration for our company. Did you build it as a separate service and use the API to sync? would love to know how you guys are doing it

**Reply by mezbahalam** [reply]
> @toobulkeh curious about this, i'm thinking about setting up a linkedin integration for our company. Did you build it as a separate service and use the API to sync? would love to know how you guys are doing it

@vincentmcleese  when you say LinkedIn integration, do you mean an official LinkedIn API-based sync (contacts/activities) or something broader like enrichment and lead import?

**Reply by toobulkeh** [reply]
unfortunately as a company we decided not to go open source with it, as ultimately it's against the TOS of LinkedIn.

**Reply by toobulkeh** [reply]
just a quick update -- we decided to MIT OSS LinkedIn messaging: https://allman.sh

**Comment by vincentmcleese** [comment]
I haven’t properly looked at the linked api but it would probably be easier
to go enrichment or import to begin with.


Vincent J. McLeese

**Comment by jevtih-svg** [comment]
Yes please, this would massively help with outreach type of business
